website/docs/d/organizations.html.markdown

```diff
@@ -27,3 +27,8 @@ The following attributes are exported:
 
 * `names` - A list of names of every organization.
 * `ids` - A map of organization names and their IDs.
+* `admin` - A boolean field that determines  the list of organizations that should
+  be retrieved. If it is true, then the [Admin Organizations
+endpoint](https://www.terraform.io/docs/cloud/api/admin/organizations.html#list-all-organizations)
+  will be used, otherwise if it is false or not included, then the [Organizations
+endpoint](https://www.terraform.io/docs/cloud/api/organizations.html#list-organizations) will be used.
```

Comment:
![image](https://user-images.githubusercontent.com/4130121/121560206-4c50f580-c9e5-11eb-95cb-c9254b0ed88b.png)

Reply:
* This should be an argument (listed above) rather than an attribute reference here. It's a user-set thing, not a piece of data retrieved. 

* I'd probably explain the difference in usage really; the user cares more about when you'd use this or not rather than which endpoint it uses (which is obvious to us); you could just add this info to what you already have, if you think the endpoint matters. You could also indicate that this is really Terraform Enterprise only, practically speaking.

> Terraform Enterprise only. If true, then the Admin Organizations endpoint will be used and the list will contain all the organization names across a Terraform Enterprise installation. If false, the Organizations endpoint will be used and all organizations readable by the token provided will be listed. Defaults to false.

Reply:
Updated
![image](https://user-images.githubusercontent.com/4130121/121610445-0f582380-ca24-11eb-80a4-abd935a47566.png)